blacktie/jatmibroker-xatmi/src/main/java/org/jboss/narayana/blacktie/jatmibroker/core/transport/OrbManagement.java

```diff
@@ -26,9 +26,9 @@
 public class OrbManagement implements Runnable {
     private static final Logger log = LogManager.getLogger(OrbManagement.class);
     private static final String CorbaOrbClassProp = "org.omg.CORBA.ORBClass";
-    private static final String CorbaOrbClassValue = "org.jacorb.orb.ORB";
+    private static final String CorbaOrbClassValue = "com.sun.corba.se.impl.orb.ORBImpl";
```

Comment:
But if you do decide to remove CORBA I think your commit which replaces JacORB with JDK orb should be kept just in case we ever do need to go back to using CORBA/OTS as the transport.